### ISSUE TITLE
Headless: setBallPosition(x, y) and setPlayerPosition(id, x, y)

### ISSUE THREAD

**Issue by tohey** [haxball/haxball-issues#425]
This would be useful for implementing a lot of things like extra time, in-game penalty kicks, throw-ins etc. Please do it! :)

**Comment by olegmisar** [comment]
Yeah, I also think it is useful. But you should add also velocity setter(and velocity getter too!). So, we will be able to reset velocity(making penalty kicks, ...) and don't change velocity when just want to teleport ball(make portals from one side of stadium to another, ...). 
```js
room.setBallVelocity(x, y)
room.setPlayerVelocity(id, x, y)
```
Or merge two functions (I don't know how to name them):
```js
room.setBall(x, y, velX, velY)
room.setPlayer(id, x, y, velX, velY)
```

**Comment by olegmisar** [comment]
Also if you want to implement these functions you should make protection against cheaters. Some "smart" programmers can abuse these functions. For example you can change the color of the ball when it's teleported.

**Comment by Leo-MapCreator** [comment]
How do you change the color of the ball while the game is running?

**Comment by olegmisar** [comment]
If you will send messages for every position set, chat will be full of this messages. Visual notification (disc blinking) will be better option.
